main.tf

```diff
@@ -2182,13 +2182,14 @@ module "external_secrets" {
 
 locals {
   fargate_fluentbit_policy_name = try(var.fargate_fluentbit_cw_log_group.create, true) ? try(var.fargate_fluentbit.policy_name, "${var.cluster_name}-fargate-fluentbit-logs") : null
+  fargate_fluentbit_cw_log_group_name = try(var.fargate_fluentbit_cw_log_group.create, true) ? try(var.fargate_fluentbit_cw_log_group.name, "/${var.cluster_name}/fargate-fluentbit-logs") : null
```

Comment:
```suggestion
  fargate_fluentbit_cw_log_group_name = try(var.fargate_fluentbit_cw_log_group.name, "/${var.cluster_name}/fargate-fluentbit-logs")
```

Reply:
Ah, we don't need it since we track it in another var, commited